src/main/java/mate/academy/hibernate/relations/dao/impl/ActorDaoImpl.java

```diff
@@ -12,11 +15,38 @@ public ActorDaoImpl(SessionFactory sessionFactory) {
 
     @Override
     public Actor add(Actor actor) {
-        return null;
+        Session session = null;
+        Transaction transaction = null;
+        try {
+            session = factory.openSession();
+            transaction = session.beginTransaction();
+            try {
```

Comment:
Is second try-catch block necessary?

Reply:
It is not necessary, but I added it to catch specific exceptions to separate two different situations when you can't add something to the database or just unable to open a session.